p2p/connmgr.go

```diff
@@ -19,63 +19,99 @@ package p2p
-func (_ ConnManager) UntagPeer(peer.ID, string)                {}
-func (_ ConnManager) UpsertTag(peer.ID, string, func(int) int) {}
-func (_ ConnManager) GetTagInfo(peer.ID) *connmgr.TagInfo      { return &connmgr.TagInfo{} }
-func (_ ConnManager) TrimOpenConns(ctx context.Context)        {}
-func (_ ConnManager) Protect(peer.ID, string)                  {}
-func (_ ConnManager) Unprotect(peer.ID, string) bool           { return false }
-func (_ ConnManager) Close() error                             { return nil }
-
-func Connected(n net.Network, c net.Conn) {
-	// TODO: replace dummy status message with current state
-	status := &StatusMessage{
-		ProtocolVersion:     0,
-		MinSupportedVersion: 0,
-		Roles:               0,
-		BestBlockNumber:     0,
-		BestBlockHash:       common.Hash{0x00},
-		GenesisHash:         common.Hash{0x00},
-		ChainStatus:         []byte{0},
-	}
-	log.Info("connected", "status", status)
+func (_ *ConnManager) TagPeer(peer.ID, string, int)             {}
+func (_ *ConnManager) UntagPeer(peer.ID, string)                {}
+func (_ *ConnManager) UpsertTag(peer.ID, string, func(int) int) {}
+func (_ *ConnManager) GetTagInfo(peer.ID) *connmgr.TagInfo      { return &connmgr.TagInfo{} }
+func (_ *ConnManager) TrimOpenConns(ctx context.Context)        {}
+func (_ *ConnManager) Protect(peer.ID, string)                  {}
+func (_ *ConnManager) Unprotect(peer.ID, string) bool           { return false }
+func (_ *ConnManager) Close() error                             { return nil }
+
+// Listen is called when network starts listening on an address
+func (cm *ConnManager) Listen(n network.Network, address ma.Multiaddr) {
+	log.Debug(
+		"started listening",
+		"host", n.LocalPeer(),
+		"address", address,
+	)
 }
 
-func OpenedStream(n net.Network, s net.Stream) {
-	if s.Protocol() == DefaultProtocolId {
-		log.Info("opened stream", "peer", s.Conn().RemotePeer(), "protocol", s.Protocol())
-	}
+// ListenClose is called when network stops listening on an address
+func (cm *ConnManager) ListenClose(n network.Network, address ma.Multiaddr) {
+	log.Debug(
+		"stopped listening",
+		"host", n.LocalPeer(),
+		"address", address,
+	)
 }
 
-func ClosedStream(n net.Network, s net.Stream) {
-	if s.Protocol() == DefaultProtocolId {
-		log.Info("closed stream", "peer", s.Conn().RemotePeer(), "protocol", s.Protocol())
+// Connected is called when a connection opened
+func (cm *ConnManager) Connected(n network.Network, c network.Conn) {
+	log.Debug(
+		"connected",
+		"host", c.LocalPeer(),
+		"peer", c.RemotePeer(),
+	)
+}
+
+// Disconnected is called when a connection closed
+func (cm *ConnManager) Disconnected(n network.Network, c network.Conn) {
+	log.Debug(
+		"disconnected",
+		"host", c.LocalPeer(),
+		"peer", c.RemotePeer(),
+	)
+}
+
+// OpenedStream is called when a stream opened
+func (cm *ConnManager) OpenedStream(n network.Network, s network.Stream) {
+	protocol := s.Protocol()
+	if protocol != "" {
```

Comment:
There are a number of streams with an empty protocol id being logged. This will only log opened streams with a protocol id. Essentially this cleans up the logs for debugging purposes.

Reply:
I don't think that's sufficient reason not to log them. You are hiding information this might be relevant to debugging.

Reply:
This is true. I think maybe just something I was using at the time. Fixed.